vertical-pod-autoscaler/pkg/recommender/main.go

```diff
@@ -72,6 +74,10 @@ var (
 	username            = flag.String("username", "", "The username used in the prometheus server basic auth")
 	password            = flag.String("password", "", "The password used in the prometheus server basic auth")
 	memorySaver         = flag.Bool("memory-saver", false, `If true, only track pods which have an associated VPA`)
+	authHeader          = flag.Bool("auth-header", false, `If True will use header authentication, If true aslo authHeaderName and authHeaderValueName must be supplied`)
+	authHeaderName      = flag.String("auth-header-name", "", `Header name to use for header based Auth`)
+	authHeaderValueName = flag.String("auth-header-value-name", "", `Env variable name holding the auth header value e.g the value of the bearer token etc..`)
```

Comment:
I think it's not clear from the flag name that an env variable key is expected here.
Could we either clarify in the flag name that this needs to be the key of an env var, or alternatively you could when reading the flag to an envsubstr and allow users to set the value to something like `${MY_ENV_VAR}`.
Either way is fine with me, what does @sophieliu15 think?

Reply:
@kwiesmueller tnx for the clarification. Updated the flag name to be more meaningful